lib/ui/semantics.dart

```diff
@@ -928,6 +943,7 @@ base class _NativeSemanticsUpdateBuilder extends NativeFieldWrapperClass1 implem
       childrenInTraversalOrder,
       childrenInHitTestOrder,
       additionalActions,
+      headingLevel ?? -1,
```

Comment:
why make headingLevel not nullable and default to -1 directly in framework??

Reply:
@chunhtai, right, I guess I made it nullable to make it easier to sync all changes between framework and engine. I just changed it to not nullable.